java-client/build.gradle.kts

```diff
@@ -146,15 +146,20 @@ val integrationTest = task<Test>("integrationTest") {
             System.getProperty("tests.awsSdk2support.domainRegion", "us-east-1"))
 }
 
+val opensearchVersion = "2.12.0-SNAPSHOT"
```

Comment:
@dblock @VachaShah we may not be able to release the new client (with JDK 8 support) till OpenSearch 2.12.0 release, where the RestClient is also JDK-8 compatible

Reply:
Makes sense.